core/src/main/java/hudson/FilePath.java

```diff
@@ -3308,8 +3308,9 @@ private void readObject(ObjectInputStream ois) throws IOException, ClassNotFound
 
     private static final long serialVersionUID = 1L;
 
-    @SuppressFBWarnings(value = "MS_SHOULD_BE_FINAL", justification = "TODO needs triage")
-    public static int SIDE_BUFFER_SIZE = 1024;
+    @Restricted(NoExternalUse.class)
+    @RestrictedSince("TODO")
+    public static final int SIDE_BUFFER_SIZE = 1024;
```

Comment:
I searched for usages in sources (with GitHub) and binaries (with `usage-in-plugins`) and could not find any references in the `jenkinsci` GitHub organization or in plugin binaries, so this is safe to restrict. Since it is restricted, it is safe to mark it final, as no plugins will inline the value during compilation.